### ISSUE TITLE
Type error

### ISSUE THREAD

**Issue by ivan-kleshnin** [rrdelaney/horizon-devtools#1]
I'm getting error after GitHub authentication for nor visible reason.

**Uncaught TypeError: Cannot read property 'replace' of undefined**

I dunno how to copy-paste full stack log from either Chrome or FF so I'll give a last fragment.

```js
var ast2string = function ast2string(ast, live) {
  return ['find', 'find_all', 'above', 'below', 'order', 'limit'].reduce(function (res, key) {
    return res + (!ast[key] ? '' : '.' + key + '(' + JSON.stringify(ast[key][0]).replace(/[\[\]\"\{\}]/g, '') + ')');
  }, (live === 'watch' ? '🔄 ' : '') + ast.collection);
};
```

Line 3: `ast[key][0]` is undefined

![shot](https://cloud.githubusercontent.com/assets/2128182/18051686/cb539a4c-6dfe-11e6-9fce-c4e78e35116f.png)

As soon as I comment

```js
let DevTools = createDevTools(horizon)
```

and

```js
export let DevTools = createDevTools(horizon)
```

lines – error disappears.

---

Error is confirmed in both Chrome and Firefox.

**Comment by rrdelaney** [comment]
What version of horizon are you using?

**Comment by ivan-kleshnin** [comment]
2.0

**Comment by rrdelaney** [comment]
Specifically what RC? I have been having problems with 2.0, and some RC's break and some don't. RC 5 worked for me, but 6 did not.

I can try to look at this, but I will have to migrate my test repo to use 2.0

**Comment by ivan-kleshnin** [comment]
I've tried with 2.0.0 which was released 5 days go.

**Comment by rrdelaney** [comment]
I didn't realize 2.0 was released. I'll update this with support soon

**Comment by rrdelaney** [comment]
Issue is fixed. There was a problem when trying to produce a readable version of find queries.

I'll publish 1.0.1 once I get access to wifi.

**Comment by ivan-kleshnin** [comment]
Great!

**Comment by rrdelaney** [comment]
Published as version 1.0.1
